config/custom-environment-variables.yml

```diff
@@ -38,3 +38,7 @@ public:
     sessionReplaySampleRate: 'DATADOG_SESSION_REPLAY_SAMPLE_RATE'
     sessionSampleRate: 'DATADOG_SESSION_SAMPLE_RATE'
     site: 'DATADOG_SITE'
+  default_otp:
+    enabled: 'DEFAULT_OTP_ENABLED'
+    code: 'DEFAULT_OTP_CODE'
+    whitelisted_phone_number_with_country_code: 'DEFAULT_OTP_WHITELISTED_NUMBERS_WITH_COUNTRY_CODE'
```

Comment:
Can you leave an example in comment as well? Some folks might do +919...., or 91-970..., we want to inform that we do not want numbers to have any special characters

Reply:
Should `whitelisted_phone_number_with_country_code` be `whitelisted_phone_numbers_with_country_code` i..e number be numbers?